ONSdigital/census-rm-docker-dev
# Add DLQ exchange & bindings for queues

# Motivation and Context

We need to make RabbitMQ responsible for routing bad messages to a DLQ instead of manually doing this in our application code. This PR updates all of the queues for devs.

# What has changed



Updated existing Rabbit queue definitions with dead letter exchange and bindings.

# How to test?



* Checkout branch `rabbit-msg-persistence-dlq-config` for the following repos (and build for each):
    * case-processor
    * action-scheduler
    * notify-service
    * fieldwork-adaptor

* `make up`
* Post bad messages directly against the rabbit queues and check that the same message is requeued continuously every 15 seconds.

# Links



https://trello.com/c/hG9Mi8tA/315-rabbit-message-persistence-dlq-config-8
